### ISSUE TITLE
[Card - pictogram] Pictogram color not using token, not accessible in dark themes

### ISSUE THREAD

**Issue by shixiedesign** [carbon-design-system/carbon-for-ibm-dotcom#6959]
Looks like pictogram's color is locked to gray100, and not changing according to the theme. This makes the pictogram unaccessible in dark themes. https://ibmdotcom-web-components-canary.mybluemix.net/?path=/story/components-card--pictogram

![image.png](https://images.zenhubusercontent.com/5be09c3a6102900c6d16d991/09d9889f-4849-4d6e-a3a6-6a903baa91ce)
